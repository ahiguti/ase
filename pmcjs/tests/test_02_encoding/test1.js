
var txt = "あいうえお";
for (var i = 0; i < txt.length; ++i) {
  print(txt.charCodeAt(i).toString(16));
}

// for (var i = 0; i < 5; ++i) alert("あいうえお".charCodeAt(i).toString(16));

