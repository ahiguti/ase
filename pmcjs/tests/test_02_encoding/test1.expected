3042
3044
3046
3048
304a
